src/client/driver/storage.js

```diff
@@ -0,0 +1,41 @@
+import hammerhead from './deps/hammerhead';
+
+var JSON = hammerhead.json;
+
+const STORAGE_KEY_PREFIX = 'testcafe|driver|';
+
+export default class Storage {
+    constructor (window, testRunId) {
+        this.storage    = window.sessionStorage;
```

Comment:
We have discussed this some time ago. Decided to implement API for this (but I dont' see an issue for this). For now we have to use just `window.sessionStorage`. Let's discuss this again when @churkin returns